source/fileio/read_network.py

```diff
@@ -276,7 +283,14 @@ def read(self, flownetwork):
         flownetwork.boundary_type = boundary_types[np.logical_or(boundary_types == 1, boundary_types == 2)]
         boundary_values = np.array(graph.vs[self._PARAMETERS["ig_boundary_value"]])
         flownetwork.boundary_val = boundary_values[np.logical_or(boundary_types == 1, boundary_types == 2)]
-        flownetwork.boundary_vs = np.arange(flownetwork.nr_of_vs)[np.logical_or(boundary_types == 1, boundary_types == 2)]
+        flownetwork.boundary_vs = np.arange(flownetwork.nr_of_vs)[
+            np.logical_or(boundary_types == 1, boundary_types == 2)]
+
+        # print(f"Network {self._PARAMETERS['network_name']}")
+        # print("Number of  vs: " + str(flownetwork.nr_of_vs))
+        # print("Number of boundary vs: " + str(len(flownetwork.boundary_vs)))
+        # print("Number of  es: " + str(flownetwork.nr_of_es))
+        #
```

Comment:
Same here! Can you please remove the lines 289-293 (I guess they are there just for you, right)?